lib/elixir/lib/file.ex

```diff
@@ -96,7 +101,24 @@ defmodule File do
   end
 
   @doc """
-  Returns `true` if the path is a directory.
+  Returns `true` if the given path is a directory.
+
+  ## Examples
+
+      File.dir("./test")
+      #=> true
+
+      File.dir("test")
+      #=> true
+
+      File.dir("/usr/bin")
+      #=> true
+
+      File.dir("~/Downloads")
+      #=> false
+
+      "~/Downloads" |> Path.expand |> File.dir?
+      #=> true
```

Comment:
Nitpick, sorry :), but we usually leave an empty line after code blocks in documentation. So basically just an empty line after this one. :)